src/planscape/planning/filters.py

```diff
@@ -0,0 +1,36 @@
+from django_filters import rest_framework as filters
+from planning.models import PlanningArea, Scenario, RegionChoices
+
+
+class PlanningAreaFilter(filters.FilterSet):
+    name = filters.CharFilter(lookup_expr="icontains")
+    region_name = filters.CharFilter(field_name="region_name", lookup_expr="in")
+    sortby = filters.CharFilter(method="filter_by_sortby")
+
+    class Meta:
+        model = PlanningArea
+        fields = ["name", "region_name"]
+
+    def filter_queryset(self, queryset):
```

Comment:
I wasn't sure if this is the right way to do it, but our spec lets users search by region with a checkbox, which I think means we'll see something like `?region_name=sierra-nevada,central-coast`. The default filters weren't handling that correctly without an override. Is there a better way to handle it? 
I agree that I'd rather not override this method.

Reply:
Yes, that's what I had before, and all of my tests with strings like `region_name=sierra-nevada,central-coast` were failing because it was trying to validate the full string with commas (ie., it was saying "sierra-nevada,central-coast" isn't a valid option).
I'm surprised there's not a built-in way to do this, but I'll try two more things here.